demisto_sdk/commands/prepare_content/tests/yml_unifier_test.py

```diff
@@ -561,33 +561,77 @@ def mocked_get_api_module_code(*args, **kwargs):
     )
 
 
-def test_insert_pack_version_and_script_to_yml():
+def test_insert_pack_version_and_script_to_yml_js_and_ps1():
     """
     Given:
-     - A pack name.
+     - A pack version.
 
     When:
-     - calling insert_pack_version.
+     - Calling insert_pack_version for different script types (.ps1, .js).
 
     Then:
-     - Ensure the code returned contains the pack version in it.
+     - Ensure the code returned contains the correct pack version.
+     - Ensure the pack version is added only once.
+     - Ensure unsupported script types do not get the pack version added.
     """
-    version_str = "### pack version: 1.0.3"
-    assert version_str not in DUMMY_SCRIPT
-    assert version_str in IntegrationScriptUnifier.insert_pack_version(
-        ".py", DUMMY_SCRIPT, "1.0.3"
-    )
-    assert version_str in IntegrationScriptUnifier.insert_pack_version(
-        ".ps1", DUMMY_SCRIPT, "1.0.3"
+    pack_version = "1.0.3"
+    version_str_ps1 = f"### pack version: {pack_version}"
+    version_str_js = f"// pack version: {pack_version}"
```

Comment:
`assert f"### pack version: {pack_version}" in updated_script_js` is clearer imo